src/localStorage.js

```diff
@@ -0,0 +1,20 @@
+export const loadState = () => {
```

Comment:
I think this file should be moved to `services` or `lib` folder. The idea is to have `components` folder where we put the React components. Generic js modules should be in their own folder and not pollute the root `src` folder in which ideally we should only have `index.js`.